standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -87,6 +87,11 @@ class CompactionTxnHandler extends TxnHandler {
       "DELETE FROM \"COMPACTION_METRICS_CACHE\" WHERE \"CMC_DATABASE\" = ? AND \"CMC_TABLE\" = ? " +
           "AND \"CMC_METRIC_TYPE\" = ?";
 
+  private static final String DELETE_FAILED_TXNS_DIRECTLY_SQL =
```

Comment:
move it to TxnQueries

Reply:
moving to TxnQueries again need to add mvn dependency. As this is different module.  
For simple we will keep in this file only as some existed SQL strings also present in this file.

Reply:
they are in the same module and package:
````
package org.apache.hadoop.hive.metastore.txn
class TxnQueries
class CompactionTxnHandler
````

Reply:
as per our discussion this we will keep in CompactionTxnHandler file.